energyplus_regressions/diffs/table_diff.py

```diff
@@ -250,7 +238,56 @@ def table2hdict_horder(table):
         hdict[hcontents] = []
         horder.append(hcontents)
 
-    for trow in trows[1:]:
+    # Assume we are going to just loop over the rows and compare the data
+    search_rows = trows[1:]
+
+    # But we can handle it specially if we passed in table_a and it's just a valid reorder
```

Comment:
Alright, this comment should capture the intent.  But basically, if we read table a, then pass it in when we read table b, this code will try to identify matching lines in the table, even if they are out of order.  If anything goes awry, diffs are detected, etc., it will revert back to the normal row-by-row comparison on both table a and table b.